packages/opencrvs/test/Adaptor.test.js

```diff
@@ -0,0 +1,196 @@
+import { expect } from 'chai';
+import { enableMockClient } from '@openfn/language-common/util';
+
+import {
+  execute,
+  request,
+  post,
+  createBirthNotification,
+  queryEvents,
+} from '../src/Adaptor.js';
+import { birthRecordData } from './testData.js';
+
+const authUrl = enableMockClient('https://auth.fake.opencrvs.server.com');
+const baseUrl = 'https://gateway.fake.opencrvs.server.com';
+const testServer = enableMockClient(baseUrl);
+
+const state = {
+  configuration: {
+    domain: 'gateway.fake.opencrvs.server.com',
+    clientId: 'someclientid',
+    clientSecret: 'someclientsecret',
+  },
+};
+
+before(() => {
+  state.configuration.domain = 'fake.opencrvs.server.com';
```

Comment:
This is strange - why are we setting the domain here, when we've defined a state object up at the top of the file? The two values are fighting each other - which one do we want?